test/msw-api/setup-server/scenarios/concurrency/process-one.test.ts

```diff
@@ -0,0 +1,89 @@
+import { rest, graphql } from 'msw'
+import { setupServer } from 'msw/node'
+import { makeRequest } from './api'
+import {
+  sharedServer,
+  sleep,
+  endpoint,
+  getNow,
+  beforeAllResolve,
+  beforeAllPromise,
+  sleepDelay,
+} from './shared-server'
+
+describe('concurrency test', () => {
+  beforeAll(async () => {
+    sharedServer.listen()
+    beforeAllResolve()
+  })
+
+  afterAll(async () => {
+    sharedServer.close()
+  })
+
+  it.concurrent('process one tests', async () => {
+    await beforeAllPromise
+
+    const mockFn = jest.fn()
+
+    console.log('step 1 - process one attach handler to server: '),
+      sharedServer.use(
+        rest.get(`${endpoint}/:processId`, (req, res, ctx) => {
+          console.log(
+            'step 4 - process one handler resolver triggered: ',
+            getNow(),
+          )
+
+          mockFn(req.params.processId)
+
+          return res(
+            ctx.json({
+              text: 'process one handler resolver response',
+            }),
+          )
+        }),
+      )
+
+    const handlers = sharedServer.listHandlers()
```

Comment:
Oh, so you want to assert the list of active handlers as well? I think that's smart. It makes this change dependent on #1369, which is okay. Now I see why you have the `listHandlers` test.